openedx/core/djangoapps/course_groups/models.py

```diff
@@ -49,3 +49,34 @@ class CourseUserGroupPartitionGroup(models.Model):
     )
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+
+class CourseCohortsSettings(models.Model):
+    """
+    This model represents cohort settings for courses.
+    """
+    is_cohorted = models.BooleanField(default=False)
+
+    course_id = CourseKeyField(
+        unique=True,
+        max_length=255,
+        db_index=True,
+        help_text="Which course is these settings associated with?",
```

Comment:
Not sure if this help text is visible in the UI, but it should be "Which course *are* these settings associated with?"